### ISSUE TITLE
Server error when attempting to download SHAKE algorithms

### ISSUE THREAD

**Issue by jbarnesweb** [usnistgov/ACVP-Server#111]
**environment**
Demo

**testSessionId**
181373

**vsId**
580804 and 580805

**Algorithm registration**

```
      {
        "algorithm": "SHAKE-128",
        "revision": "1.0",
        "messageLength": [
          {
            "max": 65535,
            "min": 0,
            "increment": 1
          }
        ]
      },
      {
        "algorithm": "SHAKE-256",
        "revision": "1.0",
        "messageLength": [
          {
            "max": 65535,
            "min": 0,
            "increment": 1
          }
        ]
      },


```

**Endpoint in which the error is experienced**
https://demo.acvts.nist.gov/acvp/v1/testSessions/181373/vectorSets/580804
https://demo.acvts.nist.gov/acvp/v1/testSessions/181373/vectorSets/580805

**Expected behavior**
Vector sets downloaded

**Additional context**
Server response:
[{"acvVersion\":"1.0"},{"error":"General exception. Contact service provider."}]

**Comment by Kritner** [comment]
Your registration is missing an `outputLen` domain, but our validation process should have caught that at registration time of your capabilities.  Will put in a fix.

**Comment by jbarnesweb** [comment]
Thank you!

**Comment by Kritner** [comment]
This change has been released to prod in release [v1.1.0.18](https://github.com/usnistgov/ACVP-Server/releases/tag/v1.1.0.18).
